docs/paper_JOSS/paper.bib

```diff
@@ -66,7 +66,7 @@ @article{scikit_learn
 
 @misc{keras,
     title={Keras},
-    author={Chollet, Fran\c{c}ois and others},
+    author={Chollet, Francois and others},
```

Comment:
this should not be corrected

Reply:
unfortunately the `\c{c}` seems to be splitting the word into two and ois is a problem...

Reply:
The other option is to get a french keyboard and type the letter with the key that has this character. Or copy it from here: ç

Reply:
I think the tool should exclude this because the latex notation is correct here.

Reply:
Yes, I am in touch with issues with resolving a few other cases... but still better than nothing :)

Reply:
```suggestion
    author={Chollet, François and others},
```